ember-client/js/src/main/scala/org/http4s/ember/client/internal/ClientHelpersPlatform.scala

```diff
@@ -27,6 +27,8 @@ import com.comcast.ip4s.IDN
 
 private[internal] trait ClientHelpersPlatform {
 
+  private[internal] type ClosedChannelException = fs2.io.IOException
```

Comment:
TIL `fs2.io.IOException`

Reply:
😆 FYI it's actually just an alias to `java.io.IOException` which _is_ supported on JS, but it's the parent to a bunch of `fs2.io`-only implementations of `java.io` exceptions for Scala.js.

Reply:
Yeah, I was just wondering whether fs2 should have a `ChannelClosedException`.

Reply:
Hmm, why/why not?

Reply:
If that's the parallel hierarchy for JavaScript implementations of IO failures.  Note that this one is specific about what went wrong, but the JavaScript will just be a generic IOException.

Reply:
Ah, yes, we can definitely add it. 👍